pkgs/top-level/python-packages.nix

```diff
@@ -27251,6 +27251,29 @@ in modules // {
     };
   };
 
+  Lasagne = buildPythonPackage rec {
+    name = "Lasagne-${version}";
+    version = "v0.1";
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "https://github.com/Lasagne/Lasagne/archive/master.zip";
```

Comment:
the HEAD of master changes over time, and therefore also the contents of this archive. 
You will have to point to an archive of which the content is (supposedly) fixed.

This package is on PyPI so please use that as source.
https://pypi.python.org/pypi/Lasagne/0.1